packages/device_info/lib/device_info.dart

```diff
@@ -241,6 +245,45 @@ class IosDeviceInfo {
       localizedModel: json['localizedModel'],
       identifierForVendor: json['identifierForVendor'],
       isPhysicalDevice: json['isPhysicalDevice'] == 'true',
+      utsname: IosUTSName._fromJson(json['utsname']),
+    );
+  }
+}
+
+/// Information derived from `utsname`.
+/// See http://pubs.opengroup.org/onlinepubs/7908799/xsh/sysutsname.h.html for details.
+class IosUTSName {
```

Comment:
@mit-mit 

nit: we should probably be consistent with the field and the Unix name `utsname` and just do `IosUtsname`. However, if we want to consider the name as made of abbreviations, then the style guide suggests `IosUtsName` (see https://www.dartlang.org/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words).

Reply:
Fixed